src/partials/hero.html

```diff
@@ -0,0 +1,22 @@
+<section class="hero-background">
+    <div class="hero-container">
+        <h1 class="visually-hidden">Simply chocolate - SHOP</h1>
+        <h2 class="hero-text">Treat yourself or a loved one to<br/> our finest ingredients for a moment<br/> of pure delight!</h2>
+        <container class="hero-buttons-all">
+            <container class="hero-buttons">
```

Comment:
tutaj również div, ale może spróbujemy innym sposobem to zrobić. Zamiast container(hero-buttons) do dwóch guzików, zrób ul dla trzech i osobną klasę dla trzeciego, możesz jeszcze pokombinować zamiast klasy z nth-child. Jak nie będzie działać to możesz zostawić ten kod. Pamiętaj dać jeszcze do validatora. Miłego kodowania!

Reply:
Ok dzięki, jeszcze się pobawię ;p

Reply:
Próbowałam z listą i rzeczywiście ok wychodziło, ale gdyby odległości były stałe, a dla zmiennych musiałyby być wyśrodkowane, w sensie takie same tu i tu. Żeby odległość między lewymi przyciskami i scrollem zmieniała się w zależności od szerokości ekranu i tak musiałby być flex, a jak próbowałam scrolla przesunąć na prawo marginesami i floatem i nawet z last-child to mi się wszystkie rozjeżdzały. Narazie tu zostanę przy mojej kombinatoryce, a jak mnie wena najdzie to jeszcze z tymi listami sprobuję, bo niby kazali używać flexa w ostatecznosci ;d